### ISSUE TITLE
context.module overwritten on setup

### ISSUE THREAD

**Issue by philtoms** [nathanmacinnes/injectr#16]
I want to be able to add module.parent to my context but this object is overwritten during the setup. Changing the context.module code to the following allows this:

```
context.module = context.module || {}; 
context.module.exports = {};
```

**Comment by nathanmacinnes** [comment]
Good spot. Do you want to create a pull request to fix it?

**Comment by nathanmacinnes** [comment]
This is now resolved as of 0.5.
